templates/deployments.yaml

```
@@ -1614,13 +1614,13 @@ spec:
         {{- end }}
         volumeMounts:
         - name: st2client-config-vol
-          mountPath: /root/.st2/
+          mountPath: /home/stanley/.st2/
```

Is the switch from `/root` to `/home/stanley` covered in the docker image changes?

We need to support older docker images too. So maybe we need to pass this path in via values?

---

Apparently st2.conf is not embedded in the Docker image like I thought: https://github.com/StackStorm/st2-dockerfiles/blob/master/base/files/st2.docker.conf

And `st2.conf` is irrelevant here any way, as this is the path to the `~/.st2` that you are adjusting, not `~/.ssh`. oops.

I wonder if this needs to be based on values:
```suggestion
          {{- if .Values.st2.system_user.user == 'root' }}
          mountPath: /root/.st2/
          {{- else }}
          mountPath: /home/{{ .Values.st2.system_user.user }}/.st2/
          {{- end }}
```

The assumptions here are spelled out in Values:
https://github.com/StackStorm/stackstorm-k8s/blob/17e5fca27a38164d77160e341ce8ba8d531f50e0/values.yaml#L118-L121

We could also add a new value `st2.system_user.home`. wdyt?

---

I like the check you have for `root`, so I've just gone ahead and taken that